### PR TITLE
Translate parts of GraphEditor.js to JSX

### PR DESCRIPTION
**Pull request by aleksanb**
During react unique key logspam debugging i ended up porting this file to JSX.
The source was found to be somewhere else, but if we want this to be JSX instead now, we can!

Should be functional now.

Depends on https://github.com/ninjadev/nin/pull/440

### CODE REVIEW

**Comment by sigvef** [comment]
needs rebase

**Comment by aleksanb** [comment]
Rebased.

File: nin/frontend/app/scripts/components/editor/GraphEditor.jsx [hunk 1]
```diff
@@ -360,13 +360,25 @@ class GraphEditor extends React.Component {
...
 
-    return e('div', {ref: ref => this.container = ref}, e('svg',
-      {height: 99999, width: 99999},
-      e('g', {
-        transform: `matrix(${this.state.scale},0,0,${this.state.scale},${this.state.x},${this.state.y})`,
-      }, graphEditorNodes,
-      connections.map(connection => e(Connection, {connection, fromPath: connection.key.split('|')[0], toPath: connection.key.split('|')[1], editor: this, scale: this.state.scale, key: connection.key})),
-      )));
+    const connectionNodes = connections.map(connection =>
+      e(Connection, {
```
Comment (sigvef) [inline nin/frontend/app/scripts/components/editor/GraphEditor.jsx]: Might as well convert this to jsx too?

**Comment by aleksanb** [comment]
Updated.

**Comment by aleksanb** [comment]
Wait there's more coming.

**Comment by aleksanb** [comment]
Updated.

**Pull request merged**